### PR TITLE
Match Frame Sections to Frame Names

### PR DESCRIPTION
**Pull request by benjchristensen**
Based on feedback of people reading spec for first time, changing section names to match the frame names, and putting in the specific "Frame Type" information for each frame with their hex value.

### CODE REVIEW

**Comment by tmontgomery** [comment]
RESUME_OK might be missing its hex value.

**Comment by benjchristensen** [comment]
It was there already, but formatted differently, so I updated the formatting to match.

**Comment by benjchristensen** [comment]
@tmontgomery Does the fix I made look okay? If so, can you approve this change?

**Comment by benjchristensen** [comment]
@fiorix Some of these changes are based on feedback from you. Does this improve the clarity of the doc? Any other changes you suggest?

File: Protocol.md [hunk 1]
```diff
@@ -613,7 +613,7 @@ The general format for an extension frame is given below.
                           Depends on Extended Type...
 ```
 
-* __Frame Type__: (16) 0xFFFF for Extension Header.
+* __Frame Type__: (16) 0xFFFF
```
Comment (fiorix) [inline Protocol.md]: Is 0xFFFF correct or should it be 0x3F?

Comment (tmontgomery) [inline Protocol.md]: I think this is 0x3F

File: Protocol.md [hunk 2]
```diff
@@ -995,7 +1007,7 @@ RESUME frames MUST always use Stream ID 0 as they pertain to the connection.
 * __Last Received Server Position__: (64) The last implied position the client received from the server.
 * __First Available Client Position__: (64) The earliest position that the client can rewind back to prior to resending frames.
 
-#### Resume OK Frame
+#### RESUME_OK Frame
```
Comment (tmontgomery) [inline Protocol.md]: This should add the hex value 0x0E

**Comment by tmontgomery** [comment]
Some nits picked.

File: Protocol.md (hunk 1 above)
Comment (benjchristensen) [inline Protocol.md]: Hmm, that seems to have regressed in merges along the way. I'm positive we fixed that a few weeks back. Fixing again ... thanks for flagging it.

Comment (benjchristensen) [inline Protocol.md]: Actually, that was a different place. The other place was fixed. Not sure how this was still wrong :-) Anyhow ... fixed now.

File: Protocol.md (hunk 2 above)
Comment (benjchristensen) [inline Protocol.md]: Done.

**Comment by benjchristensen** [comment]
Changes made.

**Comment by tmontgomery** [comment]
awesome!

**Pull request merged**